src/WebJobs.Script.WebHost/Metrics/IHostMetricsProvider.cs

```diff
@@ -24,6 +24,11 @@ public interface IHostMetricsProvider
         /// </summary>
         public string FunctionGroup { get; }
 
+        /// <summary>
+        /// Initializes the provider and starts collecting metrics.
+        /// </summary>
+        public void Start();
```

Comment:
Should this change be reverted? In your initial impl. I saw external calls to this, but in the latest it's internal calls again? The metrics provider is starting itself again on specialization.

Reply:
Good catch, will remove